blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -418,6 +474,20 @@ export default async function decorate(block) {
                         <div class="use-case-tag">Use Case 2</div>
                     </div>
                 </div>
+                <div class="main-message-wrapper">
```

Comment:
@mdickson-adbe Can you comment out this section. we are planning to add this next phase 2.0

Reply:
I'll hide it with CSS for now!